src/main/java/io/kestra/plugin/surrealdb/SurrealDBConnectionInterface.java

```diff
@@ -0,0 +1,66 @@
+package io.kestra.plugin.surrealdb;
+
+import io.kestra.core.models.annotations.PluginProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Positive;
+
+public interface SurrealDBConnectionInterface {
+
+	@Schema(
+		title = "true to use TLS for connection. Default is false"
+	)
+	@PluginProperty
+	boolean isUseTls();
+
+	@Schema(
+		title = "Connection timeout. Default is 60 seconds"
+	)
+	@PluginProperty
+	@Positive
+	int getConnectionTimeout();
+
+	@Schema(
+		title = "Connection port. Default value is 8000"
+	)
+	@PluginProperty
+	@Positive
+	int getPort();
+
+	@Schema(
+		title = "Connection host"
+	)
+	@PluginProperty(dynamic = true)
+	@NotNull @NotBlank
```

Comment:
```suggestion
	@NotBlank
```

`@NotBlank`  implies `@NotNull`

Reply:
Same fort the other